### ISSUE TITLE
Remove the `tutorial/scripts` to `aiida-tutorials`

### ISSUE THREAD

**Issue by sphuber** [aiidateam/aiida_demos#16]
The tutorials are maintained in the `aiida-tutorials` repository. These reference the scripts that are currently maintained here in `tutorials/scripts`. Having these spread over two repos makes it difficult to keep in sync. The plan is to move these to `aiida-tutorials`.

**Comment by sphuber** [comment]
Mentioning @borellim and @d-tomerini that most likely we will also move the notebooks from the `tutorial` folder.

**Comment by borellim** [comment]
Ok, but then I don't get the big difference between these two repos. I suppose there was a reason for the split between text and code? If this reason has become invalid, shouldn't we just merge the repos?

**Comment by sphuber** [comment]
No you're not being dense, it is just for historical reasons. Originally, the whole tutorial, including text and scripts was maintained in a private git repo on an EPFL server. But when we started making VMs for the tutorial, we also wanted to upload the solution scripts there for reference for the users. However, doing this from the private EPFL repo wasn't the most convenient, so we decided to move them here, which already included "public example" code. Now that we have `aiida-tutorials`, which is public, the old reason no longer holds.
